homeassistant/components/asuswrt/router.py

```diff
@@ -364,14 +416,14 @@ def host(self) -> str:
         return self._api.host
 
     @property
-    def unique_id(self) -> str | None:
-        """Return router unique id."""
-        return self._entry.unique_id
+    def mac(self) -> str | None:
+        """Return router mac address."""
+        return self._api.label_mac
```

Comment:
This method does nothing but being a proxy. Why not use `self_api.label_mac` directly?

Reply:
Method removed.